pkg/invoke/invoker.go

```diff
@@ -400,7 +417,7 @@ func (i *Invoker) Agent(ctx context.Context, c kclient.WithWatch, agent *v1.Agen
 		PreviousRunName:       opt.PreviousRunName,
 		ForceNoResume:         opt.ForceNoResume,
 		GenerateName:          opt.GenerateName,
-		DefaultModel:          project.Spec.Model,
+		DefaultModel:          model,
```

Comment:
I say we simplify this and remove this `DefaultModel` field and the one on the Run's Spec. That way, we can calculate what the default model should be in one place (in the Resume method). The base case should be `types.DefaultModelAliasTypeLLM`.